### PR TITLE
Using chromium_based to support more wider browser.

### PR DESCRIPTION
**Pull request by Eric-Guo**
[Original PR](https://github.com/fnando/browser/pull/527)

### CODE REVIEW

File: app/helpers/browser_helper.rb [hunk 1]
```diff
@@ -32,7 +29,7 @@ def unsupported_browser?
   # or mobile detection
   def browser_specific_classes
     [].tap do |classes|
-      classes << "-browser-chrome" if browser.chrome?
+      classes << "-browser-chrome" if browser.chrome? || browser.chromium_based?
```
Comment (Eric-Guo) [inline app/helpers/browser_helper.rb:35]: Maybe can remove `browser.chrome?` and below `browser.edge?`, but will also break syntax meaning of class name *-browser-chrome*, so the first choose is only add 'or' condition here.

File: app/helpers/browser_helper.rb [hunk 2]
```diff
@@ -14,15 +14,12 @@ def unsupported_browser?
...
-      # Chrome versions older than a year
-      return true if browser.chrome? && version < 109
+      # Chrome/Edge based versions older than a year
+      return true if browser.chromium_based? && version < 109
 
       # Older version of safari
       return true if browser.safari? && version < 16
 
-      # Older version of EDGE
-      return true if browser.edge? && version < 109
```
Comment (oliverguenther) [inline app/helpers/browser_helper.rb:24]: Removal of this line will result in us no longer showing the warning for the non-chromium based edge versions

**Review by oliverguenther — CHANGES_REQUESTED** [review]
Thanks for your proposal. 👍  I have a minor remark

File: app/helpers/browser_helper.rb (hunk 2 above)
Comment (Eric-Guo) [reply app/helpers/browser_helper.rb:24]: Restore and update the comments

**Comment by Eric-Guo** [comment]
@oliverguenther Thanks to review this PR again?

**Review by oliverguenther — APPROVED** [review]
Thanks for the iteration 👍

**Comment by Eric-Guo** [comment]
@oliverguenther CI passed!

**Pull request merged**